recipes/workflows/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "workflows" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7092efa188bb7a046d557b91ef1b06734b67649325c366dbb0a8a0ddf8c4e367
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - enum34  # [py<34]
+    - pip
+    - python
+    - setuptools
+    - six
+    - stomp.py
```

Comment:
Are all of these actually needed at install time?

Reply:
No, don't think they are. Removed most of them.